src/bundles/CaseStudy/components/CaseStudyForm/CaseStudyForm.js

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { connect } from 'react-redux';
+import { Form, Control, Errors } from 'react-redux-form';
+
+import { required, minArrayLength } from '../../../../validators';
+
+import Layout from '../../../../shared/Layout';
+import Textarea from '../../../../shared/Textarea';
+import MultiInput from '../../../../shared/MultiInput';
+
+const Textfield = ({ name, id, htmlFor, label, model, validators, messages, description }) => (
```

Comment:
Maybe under a `shared/form` folder as its tightly coupled to the RRF (react-redux-forms) lib